app/views/events/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="centered">
+    <h1>Upcoming Events</h1>
+    <% @upcoming_events.each do |event| %>
```

Comment:
We are using `each` heavily. There is obvious repetition in `@upcoming_events` and `@past_events`

We should have an `_event.rb` partial that we will render instead.